test/integration-test-main.sh

```diff
@@ -597,6 +621,9 @@ function add_all_tests {
     add_tests test_concurrency
     add_tests test_concurrent_writes
     add_tests test_open_second_fd
+    if pidof s3fs | xargs -I {} ps -o cmd -fp {} | grep -q ensure_diskfree; then
```

Comment:
macOS Travis complains that `cmd` is not found -- is there another way to do this?

https://travis-ci.org/s3fs-fuse/s3fs-fuse/jobs/585372211#L3704